kong/conf_loader.lua

```diff
@@ -336,6 +336,38 @@ local CONF_INFERENCES = {
   cassandra_ssl = { typ = "boolean" },
   cassandra_ssl_verify = { typ = "boolean" },
   cassandra_consistency = { enum = {
+     "DISABLED",
+     "ALL",
+     "EACH_QUORUM",
+     "QUORUM",
+     "LOCAL_QUORUM",
+     "ONE",
+     "TWO",
+     "THREE",
+     "LOCAL_ONE",
+    },
+    deprecated = {
+      replacement = "cassandra_write_consistency / cassandra_read_consistency",
+      alias = function(conf)
+        if conf.cassandra_consistency ~= "DISABLED" then
```

Comment:
I don't think that we need this new enum value. By replacing the default value to `NONE`, we already prevent new instances of the config from loading it (therefore only r/w consistencies are loaded). Then, if `cassandra_consistency` is specified, we overwrite the r/w consistencies in this aliasing. The warning log will instruct users that the property is deprecated and that they should favor the new r/w properties instead.

Reply:
@thibaultcha done